public/docs/ts/latest/guide/architecture.jade

```diff
@@ -193,196 +224,208 @@ figure
 <br clear="all">
 :marked
-  [Looking back](#component-code) at the `HeroListComponent`, we see that it's just a class.
+  [Looking back at the code](#component-code) for `HeroListComponent`, we see that it's just a class.
   There is no evidence of a framework, no "Angular" in it at all.
-  
+
   In fact, it really is *just a class*. It's not a component until we *tell Angular about it*.
-  
+
   We tell Angular that `HeroListComponent` is a component by attaching **metadata** to the class.
-  
-  The easy way to attach metadata in TypeScript is with a **decorator**. 
+
+  In !{_Lang}, we attach metadata by using an **!{_decorator}**.
   Here's some metadata for `HeroListComponent`:
-+makeExample('architecture/ts/app/hero-list.component.ts', 'metadata', 'app/hero-list.component.ts (metadata)')
+
++makeExcerpt('app/hero-list.component.ts', 'metadata')
+
 :marked
-  Here we see the `@Component` decorator which (no surprise) identifies the class
-  immediately below it as a Component class.
-  
-  A decorator is a function. Decorators often have a configuration parameter. 
-  The `@Component` decorator takes a required configuration object with the
-  information Angular needs to create and present the component and its view.
-  
-  Here we see a few of the possible `@Component` configuration options:
-  
-  * `selector` - a css selector that tells Angular to create and insert an instance of this component 
-  where it finds a `<hero-list>` tag in *parent* HTML. 
-  If the template of the application shell (a Component) contained
-<div style="margin-left:30px">
-code-example(language="html").
-    &lt;hero-list>&lt;/hero-list>
-</div>  
-:marked
-  >Angular inserts an instance of the `HeroListComponent` view between those tags.
-  
-  * `templateUrl` - the address of this component's template which we showed [above](#template).
-  
-  * `directives` - an array of the Components or Directives that *this* template requires.
+  Here we see the `@Component` !{_decorator} which (no surprise) identifies the class
+  immediately below it as a component class.
+
+block ts-decorator
+  :marked
+    A decorator is a function. Decorators often have a configuration parameter.
+    The `@Component` decorator takes a required configuration object with the
+    information Angular needs to create and present the component and its view.
+
+    Here are a few of the possible `@Component` configuration options:
+
+:marked
+  - `selector`: CSS selector that tells Angular to create and insert an instance of this component
+  where it finds a `<hero-list>` tag in *parent* HTML.
+  For example, if an app's  HTML contains `<hero-list></hero-list>`, then
+  Angular inserts an instance of the `HeroListComponent` view between those tags.
+
+  - `templateUrl`: address of this component's template, which we showed [above](#templates).
+
+  - `directives`: !{_array} of the components or directives that *this* template requires.
```

Comment:
For future reference, I don't mind using the word "list" in TypeScript instead of "array" if that makes it easier for text sharing with Dart.